services/hooks/src/v1.js

```diff
@@ -354,6 +354,22 @@ builder.declare({
   await this.Hook.remove({hookGroupId, hookId}, true);
   this.publisher.hookDeleted({hookGroupId, hookId});
 
+  try {
+    await this.LastFire.query({
+      hookGroupId: req.params.hookGroupId,
+      hookId: req.params.hookId,
+    }, {
+      handler: async (lastFire) => {
+        await lastFire.remove(false, true);
+      },
+    });
+
+    this.publisher.hookLastFiresDeleted({hookGroupId, hookId});
+    // eslint-disable-next-line no-empty
+  } catch (err) {
+    // The hook may have not been ever triggered
```

Comment:
Generally ignoring *all* errors is a mistake -- in this case, it's hiding the fact that you're calling a nonexistent `hookLastFiresDeleted` method.  If there's a specific kind of error you're looking for, you can do something like
```js
} catch (err) {
  if (err.code == 'whatever') {
    // handle it
  } else {
    throw err;
  }
}
```
that way anything unexpected still gets thrown.